packages/core/stories/banner/banner.qa.stories.tsx

```diff
@@ -37,31 +42,36 @@ const ErrorBanner = () => <BasicBannerExample status={"error"} />;
 const WarningBanner = () => <BasicBannerExample status={"warning"} />;
 const SuccessBanner = () => <BasicBannerExample status={"success"} />;
 
-export const ExamplesGrid: StoryFn = () => (
-  <StackLayout gap={2}>
-    <SaltProvider applyClassesTo={"child"} density={"high"} mode={"light"}>
-      <div style={{ width: "60vw" }}>
-        <InfoBanner />
-      </div>
-    </SaltProvider>
-    <SaltProvider applyClassesTo={"child"} density={"medium"} mode={"dark"}>
-      <div style={{ width: "60vw" }}>
-        <ErrorBanner />
-      </div>
-    </SaltProvider>
-    <SaltProvider applyClassesTo={"child"} density={"low"} mode={"light"}>
-      <div style={{ width: "60vw" }}>
-        <WarningBanner />
-      </div>
-    </SaltProvider>
-    <SaltProvider applyClassesTo={"child"} density={"touch"} mode={"dark"}>
-      <div style={{ width: "60vw" }}>
-        <SuccessBanner />
-      </div>
-    </SaltProvider>
-  </StackLayout>
+export const ExamplesGrid: StoryFn = (props) => (
+  <QAContainer cols={1} itemPadding={10} height={600} width={1000} {...props}>
+    <InfoBanner />
+    <ErrorBanner />
+    <WarningBanner />
+    <SuccessBanner />
+  </QAContainer>
 );
 
 ExamplesGrid.parameters = {
   chromatic: { disableSnapshot: false },
 };
+
+export const NoStyleInjectionGrid: StoryFn<QAContainerNoStyleInjectionProps> = (
```

Comment:
Could we not automate this, and build it in to the QAContainer?

Reply:
So you don't have to change all the QA stories

Reply:
I wanted to split it out so that I could be sure that the components are 100% being styled by the CSS or vice versa